<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>browser</class>
 <widget class="QDialog" name="browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Просмотрщик отчётов</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>177</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gb_filter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>185</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>185</height>
          </size>
         </property>
         <property name="title">
          <string>Фильтрация</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rb_bydate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>18</height>
             </size>
            </property>
            <property name="text">
             <string>По дате</string>
            </property>
            <attribute name="buttonGroup">
             <string>buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="w_bydate" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>С:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="de_from">
                 <property name="minimumSize">
                  <size>
                   <width>115</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="specialValueText">
                  <string/>
                 </property>
                 <property name="accelerated">
                  <bool>false</bool>
                 </property>
                 <property name="dateTime">
                  <datetime>
                   <hour>0</hour>
                   <minute>0</minute>
                   <second>0</second>
                   <year>2000</year>
                   <month>1</month>
                   <day>1</day>
                  </datetime>
                 </property>
                 <property name="displayFormat">
                  <string>dd.MM.yyyy</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                 <property name="currentSectionIndex">
                  <number>0</number>
                 </property>
                 <property name="date">
                  <date>
                   <year>2000</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>До:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="de_to">
                 <property name="minimumSize">
                  <size>
                   <width>115</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="wrapping">
                  <bool>false</bool>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                 <property name="displayFormat">
                  <string>dd.MM.yyyy</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_byname">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>18</height>
             </size>
            </property>
            <property name="text">
             <string>По имени ПК</string>
            </property>
            <attribute name="buttonGroup">
             <string>buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>24</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="e_name"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="b_fapply">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>82</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Применить фильтр</string>
              </property>
              <property name="text">
               <string>Фильтр</string>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/images/buttons/apply.png</normaloff>:/images/buttons/apply.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="b_fcancel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>82</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Отменить фильтр</string>
              </property>
              <property name="text">
               <string>Отмена</string>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/images/buttons/cancel.png</normaloff>:/images/buttons/cancel.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_comps">
         <property name="title">
          <string>Компьютеры</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWebView" name="reportview">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_print">
       <property name="text">
        <string>Печатать</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/images/buttons/printer.png</normaloff>:/images/buttons/printer.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_help">
       <property name="text">
        <string>Помощь</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/images/buttons/help.png</normaloff>:/images/buttons/help.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_quit">
       <property name="toolTip">
        <string>Закрыть окно просмотрщика</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/images/buttons/exit.png</normaloff>:/images/buttons/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
